Desktop/Print SCU/packages.config.merge

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
yes, you are right. it is a artifact. I swear I have already deleted it. don't know why it is still here. I am struggling a lot with github forking / branching / rebaseing / history. I am so much used to server-centric version control like subversion and team foundation server.